packages/stems/src/components/Popup/types.ts

```diff
@@ -18,7 +18,7 @@ export type PopupProps = {
   /**
    * A ref to the element whose position will be used to anchor the Popup
    */
-  anchorRef: MutableRefObject<HTMLElement | null>
+  anchorRef: RefObject<HTMLElement>
```

Comment:
Updated to RefObject as it's not modified internally. A `MutableRefObject<HTMLElement | null>` can be duck-typed as a `RefObject<HTMLElement>` so there's no breaking change here.